release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -600,11 +608,15 @@ module openconfig-interfaces {
             "Not ready to pass packets and not in some test mode.";
         }
         enum TESTING {
-          //TODO: This is generally not supported as a configured
-          //admin state, though it's in the standard interfaces MIB.
-          //Consider removing it.
           description
-            "In some test mode.";
+            "The interface is should be treated as if in admin-down state for
+            control plane protocols.  In addition, while in TESTING state the
+            device should remove the interface from aggregate interfaces.
+            This state may be set by the device as part of a transaction such
+            as gNOI Link Qualification Service.";
```

Comment:
Maybe something like:

"An interface may be set into a `TESTING` state based on a qualification workflow, or internal device state - such as those that can be triggered by the gNOI Link Qualification service."

Reply:
Modified similar to what you suggested.